### PR TITLE
versions corrected in README.md file

### PR DESCRIPTION
**Pull request by b-harsh**
To avoid any confusion for the user, I have corrected the conflicts in the versions of packages in the README.md and requirements.txt  file.

To be clear I have made the changes in the README file.

### CODE REVIEW

**Comment by b-harsh** [comment]
@fruffy can u please check the PR and tell me what is the status

File: README.md [hunk 1]
```diff
@@ -277,15 +277,15 @@ For documentation building:
 `p4c` also depends on Google Protocol Buffers (Protobuf). `p4c` requires version
 3.0 or higher, so the packaged version provided in Ubuntu 20.04 **should**
 work. However, P4C typically installs its own version of Protobuf using CMake's `FetchContent` module
-(at the moment, 3.25.3). If you are experiencing issues with the Protobuf version shipped with your OS distribution, we recommend that to install Protobuf 3.25.3 from source. You can find instructions
+(at the moment, 3.20.2). If you are experiencing issues with the Protobuf version shipped with your OS distribution, we recommend that to install Protobuf 3.20.2 from source. You can find instructions
 [here](https://github.com/protocolbuffers/protobuf/blob/v3.25.3/src/README.md).
```
Comment (AdarshRawat1) [inline README.md:281]: Description mentions [ v 3.20.2 ], but the guide provided is for v 3.25.3 of Protobuf

Comment (asl) [reply README.md:281]: 3.25.3 is the correct one

File: README.md [hunk 2]
```diff
@@ -277,15 +277,15 @@ For documentation building:
...
-After cloning Protobuf and before you build, check-out version 3.25.3:
+After cloning Protobuf and before you build, check-out version 3.20.2:
```
Comment (asl) [inline README.md:282]: ditto

File: README.md [hunk 3]
```diff
@@ -277,15 +277,15 @@ For documentation building:
...
 
-`git checkout v3.25.3`
+`git checkout v3.20.2`
```
Comment (asl) [inline README.md:284]: ditto

File: README.md [hunk 4]
```diff
@@ -277,15 +277,15 @@ For documentation building:
...
 
 Please note that while all Protobuf versions newer than 3.0 should work for
 `p4c` itself, you may run into trouble with some extensions and other p4lang
-projects unless you install version 3.25.3.
+projects unless you install version 3.20.2.
```
Comment (asl) [inline README.md:288]: ditto

**Review by asl — CHANGES_REQUESTED** [review]

**Comment by b-harsh** [comment]
Okay I am making a new PR

**Comment by AdarshRawat1** [comment]
This PR can be closed, 
Follow-up PR 
- #4540


> Okay I am making a new PR 

You can always revert changes in same PR, it helps in keeping track of suggestions and reviews based on which PR is modified.

**Comment by AdarshRawat1** [comment]
This PR can be closed, 
Follow-up PR 
- #4540


> Okay I am making a new PR 

You can always revert changes in same PR by making commits to your branch. 
It helps in keeping track of suggestions and reviews based on which PR is modified.

**Comment by fruffy** [comment]
> Okay I am making a new PR

The general workflow is to make the updates within the same PR. So please make your changes here instead. I will close the other PR.

**Comment by b-harsh** [comment]
@fruffy sorry for replying late 
Okay I will follow the general way and do the changes in the same PR

**Comment by AdarshRawat1** [comment]
Can be closed now fixed in #4555

**Comment by fruffy** [comment]
Indeed, closing in favor of #4555.

**Pull request closed without merging**